src/main/java/com/snowplowanalytics/snowplow/tracker/events/ScreenView.java

```diff
@@ -23,7 +23,7 @@
 /**
  * Constructs a ScreenView event object.
  *
- * When tracked, generates an "unstructured" or "ue" event.
+ * When tracked, generates an "unstructured" or "ue" event (this was the old name for SelfDescribing events).
```

Comment:
Do we still need to call it the Unstructured event here? I know that the "ue" value in the event type hasn't changed but I don't think users care about that. I think they only want to know if it creates a self-describing event.